types/node/index.d.ts

```diff
@@ -5972,7 +5972,7 @@ declare module "stream" {
             encoding?: string;
             objectMode?: boolean;
             read?: (this: Readable, size?: number) => any;
-            destroy?: (error?: Error) => any;
+            destroy?: (error: Error | undefined, callback: (error?: Error) => void) => void;
```

Comment:
Based on https://github.com/nodejs/node/blob/v10.7.0/lib/internal/streams/destroy.js#L32 I would expect `Error | null` here and related places.

Reply:
Changing the definition to `null` causes errors in other packages expecting `undefined`:

* [amp](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/amp/amp-tests.ts#L31)
* [stream-chain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/stream-chain/stream-chain-tests.ts#L145)

Reply:
I have pushed a fix to those tests as well.